### PR TITLE
Works and about sections from Portfolio Project

### PR DESCRIPTION
**Pull request by mperezbazan**
Add Works section with displaying in a grid
Add About sections with skills in lists

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,6 +28,218 @@ <h1 class="headline-title">Hey There.<br>I’m Alex</h1>
         <li class="button social-media-button"><a href="https://medium.com/@martin.perezbazan"><img src="./images/Medium.svg" alt="Medium icon"/></a></li>
       </ul>
     </section>
+    <section class="works-container">
+      <div class="works-title-container">
+        <p class="works-title">
+          My Recent Works
+        </p>
```
Comment (Kamba56) [inline index.html]: - [ ] Kindly use a heading element (ie `hx` element => `<h2>` or `<h3>` ... or `<h5>`) to wrap the heading of this section. It is best practice for HTML semantics.

File: index.html [hunk 2]
```diff
@@ -28,6 +28,218 @@ <h1 class="headline-title">Hey There.<br>I’m Alex</h1>
...
+      <div class="works-divider-container">
+        <hr class="works-divider"/>
+      </div>
+      <article class="work-container">
+        <div class="work-image-container">
+          <img src="./images/image2.png"/>
+        </div>
+        <p class="work-title light">
+          Multi-Post Stories
+        </p>
```
Comment (Kamba56) [inline index.html]: - [ ] Please use a heading element (ie `hx` element => `<h2>` or `<h3>` ... or `<h5>`) to wrap the title of each project. It is best practice for HTML semantics.

File: index.html [hunk 3]
```diff
@@ -28,6 +28,218 @@ <h1 class="headline-title">Hey There.<br>I’m Alex</h1>
...
+        <div class="work-footer-container-dark">
+          <a class="work-button button dark">
+            See Project
+          </a>
+        </div>
+      </article>
+    </section>
+    <section class="about-container">
+      <div class="about-content">
+        <p class="about-title">About me</p>
```
Comment (Kamba56) [inline index.html]: - [ ] Kindly use a heading element (ie `hx` element => `<h2>` or `<h3>` ... or `<h5>`) to wrap the title of this section. It is best practice for HTML semantics.

File: css/style.css [hunk 4]
```diff
@@ -10,6 +10,23 @@ main {
...
+  font-size: 17px;
+  font-weight: 700;
+  height: 24px;
+  padding: 12px;
+  width: fit-content;
+}
+
 .button:hover {
   background-color: #ff6b00;
 }
```
Comment (Kamba56) [inline css/style.css]: - [ ] Please add a hover and pressed state on your buttons to match the design template. You can use the `:hover` and `:active` pseudo-classes for this.
f2

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @mperezbazan, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
- [x] Great UI design
- [x] Great commit messages
- [x] Great use of Github flow


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mperezbazan** [comment]
I did many commits trying to resolve a lighthouse's error. but it was a General Error. so I revert many times and come back with the changes

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @mperezbazan   👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**